[Workers] notifications revision (#2960)
cloudflare/cloudflare-docs: products/workers/src/content/get-started/guide.md

<Aside type="note">

> __Note:__ When pushing to your `*.workers.dev`subdomain for the first time, you may initially see [`523` errors](https://support.cloudflare.com/hc/en-us/articles/115003011431-Troubleshooting-Cloudflare-5XX-errors#523error) while DNS is propagating. It should work after a minute or so.

Reviewer: ```suggestion
When pushing to your `*.workers.dev` subdomain for the first time, you may initially see [`523` errors](https://support.cloudflare.com/hc/articles/115003011431#523error) while DNS is propagating. It should work without any errors after a minute or so.
```

--------------------------------

## 9. Turn on/off usage notifications

> Cloudflare provide two kinds of usage notifications: Workers Weekly Summary and Workers Usage Report. They are automatically enabled when you create a new free account with Workers.

Reviewer: ```suggestion
Cloudflare provides two kinds of usage notifications: Workers Weekly Summary and Workers Usage Report. They are automatically enabled when you create a new free account with Workers.
```

![workers-usage-report](./media/workers-usage-report.png)

You can turn usage notifications on or off by going to **Account Home** > **Notifications**.

![notifications-tab](./media/notifications-tab.png)

> Select **Add** button and scroll down to Workers.

Reviewer: ```suggestion
Select **Add** and scroll down to Workers.
```
